Lib/_base64.py

```diff
@@ -0,0 +1,57 @@
+"""C accelerator wrappers for originally pure-Python parts of base64."""
+
+from binascii import Error, a2b_ascii85, a2b_base85, b2a_ascii85, b2a_base85
+from base64 import _bytes_from_decode_data, bytes_types
```

Comment:
We should avoid import cycles like this, it can make refactoring in the future harder.